src/dotnet-symbol/Program.cs

```diff
@@ -99,6 +101,20 @@ public static void Main(string[] args)
                             goto usage;
                         break;
 
+                    case "--overwrite":
+                        program.Overwrite = true;
+                        break;
+
+                    case "--timeout":
```

Comment:
Maybe add the units to the flag itself. Also, should we use a double instead of an int? `TimeSpan.FromMinutes` takes one.

Reply:
The help makes it clear that it is in minutes.  I hope that is enough. I would like keep the option name simple. I'll change it to a double.